pkg/controller/manage/manage_controller.go

```diff
@@ -113,7 +128,10 @@ func NewKmeshManageController(client kubernetes.Interface, security *kmeshsecuri
 				log.Errorf("expected *corev1.Pod but got %T and %T", oldObj, newObj)
 				return
 			}
-
+			if !isPodReady(newPod) {
+				log.Debugf("%s/%s: Pod is not ready, skipping Kmesh manage enable", newPod.GetNamespace(), newPod.GetName())
```

Comment:
Maybe we could add prefix to the log to distinguish the processing of different pod event, like "pod add event: %s/%s...."